lib/main.dart

```diff
@@ -3,107 +3,113 @@ import 'package:flutter/material.dart';
 void main() => runApp(new MyApp());
 
 class MyApp extends StatelessWidget {
```

Comment:
Should we decide a name and update it in first commit itself?

Reply:
rename it to ProjectNeo. sounds good?

Reply:
Ya, that fine.